scripts/nav-tree-utils.js

```diff
@@ -0,0 +1,83 @@
+import { createOptimizedPicture } from './lib-franklin.js';
+
+function addBackdropEventListeners(element) {
+  element.addEventListener('mouseover', () => {
+    document.querySelector('.backdrop').classList.add('visible');
+  });
+  element.addEventListener('mouseleave', () => {
+    document.querySelector('.backdrop').classList.remove('visible');
+  });
+}
+
+/**
+ * @param {String} HTML representing a single element
+ * @return {Element}
+ */
+function htmlToElement(html) {
+  const template = document.createElement('template');
+  const trimmedHtml = html.trim(); // Never return a text node of whitespace as the result
+  template.innerHTML = trimmedHtml;
+  return template.content.firstChild;
+}
+
+function decorateChildNodes(parent, json, level) {
+  const nodes = json.reduce((accumalator, data) => {
+    if (data.parent && data.parent.toLowerCase() === parent.category.toLowerCase()) {
+      const children = data.hasChild === 'true' ? decorateChildNodes(data, json, level + 1) : '';
```

Comment:
hasChild needs to be removed.

Reply:
this comment i have added for myself to remove hasChild property, you can simply ignore